packages/govuk-frontend/src/govuk/components/summary-list/summary-list.yaml

```
@@ -109,12 +109,12 @@ params:
       - name: actions
         type: object
         required: false
-        description: The actions of each summary card.
+        description: The action link content for each summary card variant of the summary list component.
```

I think this could be clearer in how it differs from the other `actions` option – it does relate to the summary card, but the difference is that the actions are added to the card (within the grey area at the top) rather than to the individual rows.

The way this is currently written suggests it's an alternative way of doing the same thing, if you're using the variant, and doesn't really help the user understand what the difference is.

---

The `card` description says the "…summary card will wrap around the summary list" so how about this?

```suggestion
        description: The action link content shown in the header of each summary card wrapped around the summary list component.
```